src/Compilation.zig

```diff
@@ -2590,6 +2590,8 @@ fn updateStage1Module(comp: *Compilation, main_progress_node: *std.Progress.Node
     man.hash.add(comp.emit_llvm_ir != null);
     man.hash.add(comp.emit_analysis != null);
     man.hash.add(comp.emit_docs != null);
+    if (comp.test_filter != null) man.hash.add(comp.test_filter.?);
+    if (comp.test_name_prefix != null) man.hash.add(comp.test_name_prefix.?);
```

Comment:
```suggestion
    man.hash.addOptionalBytes(comp.test_filter);
    man.hash.addOptionalBytes(comp.test_name_prefix);
```

Reply:
Of course. Should have spotted that with zls. Thanks. Updated.